pennylane/ops/qubit.py

```diff
@@ -1707,14 +1707,13 @@ class SingleExcitation(Operation):
 
     * Number of wires: 2
     * Number of parameters: 1
-    * Gradient recipe: Obtained from its decomposition in terms of the
-      :class:`~.SingleExcitationPlus` and :class:`~.SingleExcitationMinus` operations
+    * Gradient recipe: The SingleExcitation operator satisfies a four-term parameter-shift rule
```

Comment:
@dwierichs just to confirm, this is what you had in mind for enabling both operations to use the four-term shift rules?

Reply:
yes, exactly :)